README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# _Kortney Foley's Portfolio_
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A portfolio site for Kortney Foley
 
-## Available Scripts
+---
 
-In the project directory, you can run:
+## Authors
 
-### `npm start`
+Kortney Foley
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+---
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Technologies Used
 
-### `npm test`
+-   HTML
+-   JavaScript
+-   CSS
+-   React
+-   [Scrollspy](https://www.npmjs.com/package/react-scrollspy)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+---
 
-### `npm run build`
+## Description
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+I designed this app to be a showcase of my projects and skills as a Full-Stack developer. Icons in the projects section link to a selection of other applications I've created. The page also covers skills I've developed and has a contact form at the bottom.
```

Comment:
I don't think it has a form at the bottom does it?
It definitely has contact information though